go-controller/pkg/ovn/secondary_layer2_network_controller_test.go

```diff
@@ -476,7 +495,7 @@ func dummyPrimaryLayer2UserDefinedNetwork(subnets string) secondaryNetInfo {
 	return secondaryNet
 }
 
-func dummyL2TestPod(nsName string, info secondaryNetInfo, podIdx int) testPod {
+func dummyL2TestPod(nsName string, info secondaryNetInfo, podIdx, secondaryNetIdx int) testPod {
```

Comment:
the 2 indexes thing is a bit hard to reason about ... While I think I understand what's happening, expanding on this will be increasingly harder in the future. 

Can't you abstract this some other way ?...

Reply:
problem is that the index is used to generate the ip address, so it should be the same for source and target pod for localnet, that's why we have an idex for default pod network and an index for secondary, don't know if it's worth it to abstract that away.